Livecheckables/bison@2.7.rb

```diff
@@ -0,0 +1,6 @@
+class BisonAT27
+  livecheck do
+    url "https://ftp.gnu.org/gnu/bison/?C=M&O=D"
+    regex(%r{href="bison-(2(?:\.\d+)+)\.t}i)
```

Comment:
```suggestion
    regex(/href=.*?bison-(2(?:\.\d+)+)\.t/i)
```
Just a doubt as well, the formula name seems to be `bison@2.7` so would we want to match only `2.7.x` or all `2.x.x`? The current regex matches all `2.x.x`.